CPAC/anat_preproc/anat_preproc.py

```diff
@@ -1194,7 +1172,10 @@ def freesurfer_abcd_brain_connector(wf, cfg, strat_pool, pipe_num, opt):
 
     wf.connect(binary_filled_mask, "out_file", brain_mask_to_t1_restore, "in_file")
 
-    node, out = strat_pool.get_data("desc-preproc_T1w")
+    if strat_pool.check_rpool("desc-restore_T1w"):
+        node, out = strat_pool.get_data("desc-restore_T1w")
+    else:
+        node, out = strat_pool.get_data("desc-preproc_T1w")
```

Comment:
[same thing](https://github.com/FCP-INDI/C-PAC/pull/2268#discussion_r2231623527)

```suggestion
    node, out = strat_pool.get_data(["desc-restore_T1w", "desc-preproc_T1w"])
```